test/onnx/test_fx_to_onnx_with_onnxruntime.py

```diff
@@ -1321,7 +1321,7 @@ def create_kwargs():
         )
 
     @pytorch_test_common.xfail(
-        error_message="whole graph export entails exactly one guard export"
+        error_message="Dynamic shape check failed for graph inputs"
```

Comment:
Test proceeded from the last failure and proceeded to some new failure. Its an improvement over the current state. 
Actually fixing the test is out of scope of this PR.